pages/common/scrcpy.md

```diff
@@ -7,6 +7,10 @@
 
 `scrcpy`
 
+- Display a mirror of a specific device based on its ID or IP address. You can find it under the "adb devices" command:
```

Comment:
Perhaps this will read better?
```suggestion
- Display a mirror of a specific device based on its ID or IP address (find it under the `adb devices` command):
```

Generally we try to avoid the 2nd person (e.g. `you`, `your`, etc) in tldr-pages :slightly_smiling_face:

Reply:
Thnks for you feedback. I was unsure to use the backticks on the command description, so I use "..." instead to avoid conflicts with tldr clients.